user_tools/docs/index.md

```diff
@@ -150,7 +150,8 @@ The following table summarizes the commands supported for each cloud platform:
 |                  |               +-------------+-----------------------------------------+----------+
 |                  |               | cluster     |           unsupported                   |    N/A   |
 |                  +---------------+-------------+-----------------------------------------+----------+
-|                  | profiling     | local       |           unsupported                   |    N/A   |
+|                  | profiling     | local       | spark_rapids_user_tools \               |  23.08+  |
+|                  |               |             |   databricks_aws profiling [ARGS]       |          |
```

Comment:
Sorry, should have caught this with Databricks Azure work.  But I thought platform name was uses dash and not underscore?  E.g. dataproc-t4, databricks-aws.  @amahussein

Reply:
The scala module uses "-", while python package uses "_"
It should be an easy fix because all we need to do is to change they map in `wrapper.py`

```
def main():
    fire.Fire({
        'emr': EMRWrapper,
        'dataproc': DataprocWrapper,
        'databricks-aws': DBAWSWrapper,
        'databricks-azure': DBAzureWrapper,
        'onprem': OnPremWrapper
    })
```

Reply:
Changes updated. Thank you!